src/App.js

```diff
@@ -1,5 +1,90 @@
+import { MissionUtils } from "@woowacourse/mission-utils";
+
 class App {
-  async play() {}
+
+  getRandomNumber(){
+    const computerNumber = [];
+    while (computerNumber.length < 3){
+      const number = MissionUtils.Random.pickNumberInRange(1, 9);
+      if(!computerNumber.includes(number))
+        computerNumber.push(number);
+    }
+    return computerNumber;
+  }
+
+  async getUserInput(){
+    const userInput = await MissionUtils.Console.readLineAsync("숫자를 입력해주세요 : ");
+    const validatedUserInput = this.checkInputValidation(userInput);
+
+    return validatedUserInput;
+  }
+
+  checkInputValidation(userInputNumber){
+    const userInputNumberArr = userInputNumber.split('').map(Number);
+
+    if(userInputNumber.length !== 3) throw new Error("[ERROR] 3자리 숫자가 아닙니다.");
+    if(new Set(userInputNumberArr).size !== 3) throw new Error("[ERROR] 중복된 숫자가 있습니다.");
+    if(userInputNumber.includes(0)) throw new Error("[ERROR] 1~9 사이의 숫자가 아닙니다.");
+    if(isNaN(userInputNumber)) throw new Error("[ERROR] 숫자가 아닙니다.");
+    
+    return userInputNumberArr;
+  }
+
+  compareNumber(computerNumber, userInput){
+    let strikeCount = 0;
+    let ballCount = 0;
+
+    for(let i = 0; i < 3; i++){
+      if(computerNumber[i] === userInput[i]) strikeCount++;
+      else if(computerNumber.includes(userInput[i])) ballCount++;
+    }
+    return {strikeCount, ballCount};
+  }
+
+  async answerResult(computerNumber, userInput){
+    if(computerNumber.join('') === userInput.join('')){
+      MissionUtils.Console.print("3스트라이크");
+      return true;
+    } else {
+      const STRIKE_COUNT = this.compareNumber(computerNumber,userInput).strikeCount;
+      const BALL_COUNT = this.compareNumber(computerNumber,userInput).ballCount;
```

Comment:
구조분해할당으로 한줄로 축약 가능해보입니다
또한 상수가 아닌값은 camelCase로 작성하시는게 좋아보입니다!

구조분해할당 : https://ko.javascript.info/destructuring-assignment#ref-863
에어비앤비 컨벤션 : https://github.com/airbnb/javascript#naming--camelCase

```suggestion
      const {strikeCount, ballCount} = this.compareNumber(computerNumber,userInput);
```